src/redux/mission/mission.js

```diff
@@ -47,6 +53,14 @@ const missionReducer = (state = initialState, { type, payload }) => {
         return { ...mission, reserved: true };
       });
       return newState;
+    case CANCEL_MISSION:
+      const leaveState = state.map((mission) => {
+        if (mission.mission_id !== payload) {
+          return mission;
+        }
+        return { ...mission, reserved: false };
+      });
+      return leaveState;
```

Comment:
You have your <code>CANCEL_MISSION</code> case as expected, but I can not figure out where do you save the <code>reserved: false</code> key, according to the requirements, see the picture below ⬇️

![Captura de Pantalla 2021-12-15 a la(s) 8 54 05](https://user-images.githubusercontent.com/88604987/146211533-d341f8aa-2c4f-4268-8a29-fbd38b8dee58.png)

Can you please add the key to your <code>initialState</code> ❓